js/mage/adminhtml/grid.js

```diff
@@ -56,6 +56,7 @@ varienGrid.prototype = {
 
                 Event.observe(this.rows[row],'mouseover',this.trOnMouseOver);
                 Event.observe(this.rows[row],'mouseout',this.trOnMouseOut);
+                Event.observe(this.rows[row],'mousedown',this.trOnClick);
```

Comment:
is Event a prototypejs thingy?

Reply:
Yes. See http://api.prototypejs.org/dom/Event/observe/index.html

Reply:
FYI, this is needed because middle click doesn’t fire the click event.

Reply:
mmm I'd much rather remove some prototype code instead of adding it :-\

Reply:
I would like to remove prototype too, but at some point it will be better to completely overhaul the JS code used in admin grids, because the whole file is based on `Class.create()`:

```js
var varienGrid = new Class.create();
varienGrid.prototype = {
  // ...
```

Converting a few lines in this file seems futile -- I like smaller wins where we can completely remove prototype from a certain phtml file, or completely rewrite a .js file.